internal/contributebot/process.go

```diff
@@ -160,8 +161,9 @@ func processPullRequestEvent(data *pullRequestData) *pullRequestEdits {
 	defer log.Printf("-> %v", edits)
 	pr := data.PullRequest
 
-	// If the pull request is from a branch of the main repo, close it and request that it come from a fork instead.
-	if data.Action == "opened" && pr.GetHead().GetRepo().GetName() == data.Repo {
+	// If the pull request is from a branch of the main repo, close it and request
```

Comment:
from a branch of the main repo -> not from a fork 

in the comment.

Reply:
Done